external/esp_idf_port/wpa_supplicant/include/wpa2/tls/libtommath.h

```diff
@@ -81,2667 +81,2613 @@ static const char mem_debug_file[] ICACHE_RODATA_ATTR = __FILE__;
+		 *
+		 * e.g.
+
+		 b-2 | b-1 | b0 | b1 | b2 | ... | bb |   ---->
+					 /\                   |      ---->
+					  \-------------------/      ---->
+		 */
+		for (x = 0; x < (a->used - b); x++) {
+			*bottom++ = *top++;
+		}
+
+		/* zero the top digits */
+		for (; x < a->used; x++) {
+			*bottom++ = 0;
+		}
+	}
+
+	/* remove excess digits */
+	a->used -= b;
 }
 
-
-/* swap the elements of two integers, for cases where you can't simply swap the 
+/* swap the elements of two integers, for cases where you can't simply swap the
  * mp_int pointers around
  */
 static void
-mp_exch (mp_int * a, mp_int * b)
+mp_exch(mp_int *a, mp_int *b)
 {
-  mp_int  t;
+	mp_int t;
 
-  t  = *a;
-  *a = *b;
-  *b = t;
+	t = *a;
+	*a = *b;
+	*b = t;
 }
 
-
-/* trim unused digits 
+/* trim unused digits
  *
  * This is used to ensure that leading zero digits are
  * trimed and the leading "used" digit will be non-zero
  * Typically very fast.  Also fixes the sign if there
  * are no more leading digits
  */
 static void
-mp_clamp (mp_int * a)
+mp_clamp(mp_int *a)
 {
-  /* decrease used while the most significant digit is
-   * zero.
-   */
-  while (a->used > 0 && a->dp[a->used - 1] == 0) {
-    --(a->used);
-  }
-
-  /* reset the sign flag if used == 0 */
-  if (a->used == 0) {
-    a->sign = MP_ZPOS;
-  }
+	/* decrease used while the most significant digit is
+	 * zero.
+	 */
+	while (a->used > 0 && a->dp[a->used - 1] == 0) {
+		--(a->used);
+	}
+
+	/* reset the sign flag if used == 0 */
+	if (a->used == 0) {
+		a->sign = MP_ZPOS;
+	}
 }
 
-
 /* grow as required */
 static int
-mp_grow (mp_int * a, int size)
+mp_grow(mp_int *a, int size)
 {
-  int     i;
-  mp_digit *tmp;
-
-  /* if the alloc size is smaller alloc more ram */
-  if (a->alloc < size) {
-    /* ensure there are always at least MP_PREC digits extra on top */
-    size += (MP_PREC * 2) - (size % MP_PREC);
-
-    /* reallocate the array a->dp
-     *
-     * We store the return in a temporary variable
-     * in case the operation failed we don't want
-     * to overwrite the dp member of a.
-     */
-    tmp = OPT_CAST(mp_digit) XREALLOC (a->dp, sizeof (mp_digit) * size);
-    if (tmp == NULL) {
-      /* reallocation failed but "a" is still valid [can be freed] */
-      return MP_MEM;
-    }
-
-    /* reallocation succeeded so set a->dp */
-    a->dp = tmp;
-
-    /* zero excess digits */
-    i        = a->alloc;
-    a->alloc = size;
-    for (; i < a->alloc; i++) {
-      a->dp[i] = 0;
-    }
-  }
-  return MP_OKAY;
+	int i;
+	mp_digit *tmp;
+
+	/* if the alloc size is smaller alloc more ram */
+	if (a->alloc < size) {
+		/* ensure there are always at least MP_PREC digits extra on top */
+		size += (MP_PREC * 2) - (size % MP_PREC);
+
+		/* reallocate the array a->dp
+		 *
+		 * We store the return in a temporary variable
+		 * in case the operation failed we don't want
+		 * to overwrite the dp member of a.
+		 */
+		tmp = OPT_CAST(mp_digit) XREALLOC(a->dp, sizeof(mp_digit) * size);
```

Comment:
Please remove space between `OPT_CAST(mp_digit)` and `XREALLOC`.

Reply:
@jeongchanKim 
OK - I have a question. Does erasing the space affect the code? or Just Code style?

Reply:
Not affect the operation. It is just for our coding style. Thanks.

Reply:
@jeongchanKim 
I'd appreciate it if you could check the Code style again.